content/docs/authorization/deployment/helm/_index.md

```diff
@@ -231,6 +231,11 @@ data:
 
 This secret must be applied in the driver namespace. Continue reading the next section for configuring the driver to use CSM Authorization.
 
+>__Note__:
```

Comment:
Do we need this note about jq if we're removing the jq dependency?

Reply:
Using jq should be the preferred approach as it is meant to be used for structured data verses some Unix commands to parse text. Parsing text is fragile. If I make the command more robust then it gets more complex. We should just stick with jq.

I'm not a fan of making doc changes to add substitutions as we should have documented the sed commands in the original issue and closed. Most admins should already have some idea of what they can use as an alternative, even manual copy and pasting, especially if they are managing a k8s cluster in a Linux environment.

Reply:
Should we keep the original command with jq as an option for those that have jq in their environment, as well as add this new command with sed?

Reply:
What if we had a common section where we cover the tools that are not normally in a distribution. I can see the RPM locations for the selinux as another use case.

Reply:
We could change to use a more robust Python 3 option. It is more likely that Python 3 is bundled with the distribution.

| python3 -c "import json,sys; print(json.load(sys.stdin)['Token'])"

Reply:
OK I changed to using Python. More robust than sed at a smaller risk of changes to the output.